airflow/ui/src/views/Docs.tsx

```diff
@@ -18,12 +18,160 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  Icon,
+  Link,
+  List,
+  ListItem,
+  Text,
+  useColorModeValue,
+} from '@chakra-ui/react';
+import { FaGithub } from 'react-icons/fa';
+import { FiExternalLink, FiGlobe } from 'react-icons/fi';
+
+import AppContainer from 'containers/AppContainer';
 
 const Docs: React.FC = () => (
-  <Center height="100vh">
-    <Heading>Docs</Heading>
-  </Center>
+  <AppContainer>
+    <Box mx="auto" my={8} maxWidth="900px">
+      <Flex mt={8}>
+        <Box flex="1">
+          <Heading as="h1">Documentation</Heading>
+          <Text mt={4}>
+            Apache Airflow Core, which includes webserver, scheduler, CLI and other components that
+            are needed for minimal Airflow&nbsp;installation.
+          </Text>
+          <Button
+            as="a"
+            href="https://airflow.apache.org/docs/apache-airflow/stable/index.html"
+            variant="solid"
+            rightIcon={<FiExternalLink />}
+            mt={4}
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            Apache Airflow Docs
+          </Button>
+        </Box>
+        <Box ml={8} p={4} bg={useColorModeValue('gray.100', 'gray.700')} borderRadius="md">
+          <Heading as="h3" size="sm">Links</Heading>
+          <List mt={4} spacing={2}>
+            <ListItem>
+              <Link href="https://airflow.apache.org/" isExternal color="teal.500">
+                <Icon as={FiGlobe} mr={1} />
+                Apache Airflow Website
+              </Link>
+            </ListItem>
+            <ListItem>
+              <Link href="https://github.com/apache/airflow" isExternal color="teal.500">
+                <Icon as={FaGithub} mr={1} />
+                apache/airflow on GitHub
+              </Link>
+            </ListItem>
+          </List>
+        </Box>
+      </Flex>
+      <Box mt={10}>
+        <Heading as="h3" size="lg">REST API Reference</Heading>
+        <Flex mt={4}>
+          <Button
+            as="a"
+            href="http://127.0.0.1:28080/api/v1/ui/"
+            variant="outline"
+            rightIcon={<FiExternalLink />}
+            mr={2}
+          >
+            Swagger
+          </Button>
+          <Button
+            as="a"
+            href="http://127.0.0.1:28080/redoc"
+            variant="outline"
+            rightIcon={<FiExternalLink />}
+          >
+            Redoc
+          </Button>
+        </Flex>
+      </Box>
+      <Box mt={10}>
+        <Heading as="h3" size="lg">Providers Packages</Heading>
+        <Text mt={4}>
+          Providers packages include integrations with third party integrations.
+          They are updated independently of the Apache Airflow&nbsp;core.
+        </Text>
+
+        <List spacing={2} mt={4} style={{ columns: 3 }}>
+          <ListItem><Link href="https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/index.html" isExternal color="teal.500">Amazon</Link></ListItem>
```

Comment:
Could we store all this info as an array of objects we parse through? I think it would be easier to read that way.